spec/controllers/v2/tailorings_controller_spec.rb

```diff
@@ -253,6 +247,42 @@
       end
     end
 
+    context 'with authorization only via cert_auth' do
+      before do
+        allow_any_instance_of(V2::ApplicationController).to receive(:rbac_allowed?).and_call_original
+        allow_any_instance_of(Insights::Api::Common::HostInventory).to receive(:hosts).and_return('results' => [:foo])
+        header = JSON.parse(Base64.decode64(current_user.account.identity_header.raw))
+        header['identity']['auth_type'] = Insights::Api::Common::IdentityHeader::CERT_AUTH
+        request.headers['X-RH-IDENTITY'] = Base64.encode64(header.to_json)
+      end
+
+      let(:parent) do
+        FactoryBot.create(
```

Comment:
Consider using the `for_tailoring` trait, there are examples on how to use it already.